src/windows/Windows.jsx

```diff
@@ -659,12 +659,20 @@ export default class WindowsIndex extends Component {
         </DashboardPage>
       );
     }
-
-    return (
-      <DashboardPage title="Windows" subtitle="Release Criteria Report">
-        {reduced}
-      </DashboardPage>
-    );
+    if (bits === 32) {
+      return (
+        <DashboardPage title="Windows 32" subtitle="Release Criteria Report">
+          {reduced}
+        </DashboardPage>
+      );
+    }
+    if (bits === 64) {
+      return (
+        <DashboardPage title="Windows 64" subtitle="Release Criteria Report">
+          {reduced}
+        </DashboardPage>
+      );
+    }
```

Comment:
Use the `bits` variable right tin the template, do not make `if/else` (or `if/if` in your case) blocks:

Example

    <DashboardPage title={"Windows "+ bits} ....

Reply:
Done sir. Thanks for the review.